src/extension/app/plugins/preview/preview.js

```diff
@@ -43,7 +43,8 @@ export function createPreviewPlugin(appStore) {
     button: {
       text: appStore.i18n('preview'),
       action: async () => {
-        const { status, location } = appStore;
+        const { location } = appStore;
+        const status = await appStore.fetchStatus(false, true, true);
```

Comment:
Needs to fetch status with `editUrl=auto` now when previewing

Reply:
@rofe Looking at the `preview` plugin I see we required two pieces of information in particular in order to preview. `sourceLocation` and `contentType`. Since this data does not get returned in a cheap status call on a resource that has yet to be previewed, we have to call status with `editUrl=auto` in order to get this information.

That said, you made me realize that we can be smarter about this and only call status when previewing if the information doesn't exist on the `preview` object returned in the cheap status call.

wdyt?

Reply:
Actually.. this doesn't work. The `contentType` in `preview` is `text/plain; charset=utf-8"`. So looks like we always have to make this call.